cdap-client/src/main/java/co/cask/cdap/client/ProgramClient.java

```diff
@@ -292,6 +292,48 @@ public void setFlowletInstances(String appId, String flowId, String flowletId, i
   }
 
   /**
+   * Gets the number of instances that a worker is currently running on.
+   *
+   * @param appId ID of the application that the worker belongs to
+   * @param workerId ID of the worker
+   * @return number of instances that the worker is currently running on
+   * @throws IOException if a network error occurred
+   * @throws NotFoundException if the application or worker could not be found
+   * @throws UnAuthorizedAccessTokenException if the request is not authorized successfully in the gateway server
+   */
+  public int getWorkerInstances(String appId, String workerId) throws IOException, NotFoundException,
```

Comment:
How would this work if you don't have an endpoint to get instances? ... or am I missing something?